RadixWallet/Features/ShieldSetup/PrepareFactors/Children/AddFactor+View.swift

```diff
@@ -0,0 +1,123 @@
+import SwiftUI
+
+// MARK: - PrepareFactors.AddFactor.View
+extension PrepareFactors.AddFactor {
```

Comment:
`Factor` is verboten!!! Call it `FactorSource` which is the level we are working on here.